ContentSerializer/ProgramInterfaceCLI.cs

```diff
@@ -15,19 +15,30 @@ class ProgramInterfaceCLI
     {
         
 
-        private static ProgramInterface prog = new ProgramInterface();
-        private static Dictionary<string, string> paterns = new Dictionary<string, string>
+        private ProgramInterface prog = new ProgramInterface();
+        private Dictionary<string, string> paterns = new Dictionary<string, string>
         {
             {"items", @"Data\/Items\/([^\/]+)\.inibin"},
             {"itemgroups", @"Data\/Items\/ItemGroups\/([^\/]+)\.inibin"},
             {"itemmetadata", @"Data[\/\\]Items[\/\\]metadata[\/\\](.*).inibin"},
             {"spells", @"Data(|\/Characters\/([^\/]+)|\/Shared)\/Spells\/([^\/]+)\.inibin"},
-            {"characters", @"Data\/Characters\/([^\/]+)(\/\1\.inibin)"},
+            {"char", @"Data\/Characters\/([^\/]+)(\/\1\.inibin)"},
             {"talents", @"Data\/Talents\/([^\/]+)\.inibin"},
             {"skins", @"Data\/Characters\/([^\/]+)\/Skins\/([^\/]+)\/\2\.inibin"},
             {"all", @"(.*)\.inibin"}
         };
 
+        public int LoadPatern(string filename)
```

Comment:
There is too much of "patern".
If i changed it to pattern i will need to change it commands too and i will need to adjust to change.
Whole command line interface is just patched up anyways.
Whole CLI is just there to ease getting  hashes anyways.
Main point of this PR was parsing inibin values the correct way.
Everything else was just a bonus.